sdk/cosmos/azure-cosmos/azure/cosmos/exceptions.py

```diff
@@ -55,6 +55,15 @@ def __init__(self, status_code=None, message=None, response=None, **kwargs):
 
 class CosmosResourceNotFoundError(ResourceNotFoundError, CosmosHttpResponseError):
     """An HTTP error response with status code 404."""
+    def __init__(self, status_code=None, message=None, response=None, sub_status=None, **kwargs):
+        if sub_status and not response:
+            self.http_error_message = message
+            self.sub_status = sub_status
```

Comment:
We shouldn't add a new attribute called `http_error_message` - we already have the message attribute.
Please also properly document these new attributes to the class the docstring, and fully type hint the constructor method signature.

Reply:
Nevermind - these attributes are actually already on the parent object, so leave them unaltered. Though it would be great if we could properly document these.